rsocket-core/src/main/java/io/rsocket/fragmentation/FragmentationDuplexConnection.java

```diff
@@ -72,7 +84,7 @@ public FragmentationDuplexConnection(DuplexConnection delegate, int maxFragmentS
    *     be fragmented.
    * @throws NullPointerException if {@code byteBufAllocator} or {@code delegate} are {@code null}
    * @throws IllegalArgumentException if {@code maxFragmentSize} is not {@code positive}
-   */
+   *//*
```

Comment:
Do you want to check in this commented code?

Reply:
yeah - should have added to my pull request that its not quite done yet, and there's a couple things left. I wanted people to start looking at it.

Reply:
@yschimke just removed one commented block (not this one though), so those will be cleaned out also - one change at a time.